src/types.h

```diff
@@ -531,6 +531,9 @@ struct Probe
             address,
             func_offset);
   }
+
+public:
+  std::vector<std::string> funcs;
```

Comment:
This can go w/ the other fields in `struct Probe`. Fields in a struct are public by default.

Also please add to the `archive()` call so the field gets serialized for AOT (hope to return to that soon...)

Reply:
Also please add a comment describing what this field is used for (multiattach kprobes)

Reply:
ok, will check, thanks